qiskit_experiments/library/characterization/__init__.py

```diff
@@ -97,6 +98,7 @@
 from .t2hahn import T2Hahn
 from .tphi import Tphi
 from .cr_hamiltonian import CrossResonanceHamiltonian, EchoedCrossResonanceHamiltonian
+from .cr_phase import CrossResonancePhase
```

Comment:
Because cr fine amp is in the `fine_amplitude.py` perhaps this should go to `fine_phase.py` rather than specific scope under `cr_`?